### PR TITLE
When login link is clicked on a mobile device, go to the app or the app store.

### PR DESCRIPTION
**Pull request by yipingliao**
@andrewconner 
How can I test this in dev?

https://app.asana.com/0/19898402862947/18964106149131

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/layout/header/PublicHeaderCtrl.js [hunk 1]
```diff
@@ -8,13 +8,18 @@ angular.module('kifi')
...
-      $scope.$emit('getCurrentLibrary', { callback: function (lib) {
-        var userData;
-        if (lib && lib.id) {
-          userData = { libraryId: lib.id };
-        }
-        signupService.register(userData);
-      }});
+
+      if (platformService.isSupportedMobilePlatform()) {
+        platformService.goToAppOrStore();
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/header/PublicHeaderCtrl.js:13]: Do we need a `$event.preventDefault`?

**Comment by andrewconner** [comment]
Chrome DevTools can pretend to be a phone. Happy to show you.

File: kifi-backend/modules/shoebox/angular/src/layout/header/PublicHeaderCtrl.js (hunk 1 above)
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/layout/header/PublicHeaderCtrl.js:13]: Yes, otherwise the browser will go to "/signup".

**Comment by yipingliao** [comment]
Works in dev now. Had to pull in your latest changes. Gonna merge.

**Pull request merged**